### PR TITLE
Mark syndicate reinforcement as antag

### LINKED ISSUE #17003: syndicate reinforcements are not marked as antag
## Description

**Reproduction**

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by Equivocateur**
## About the PR
Marks syndicate reinforcement agent ghost roles as antagonists, closes #17003.
Depends on #17009.

**Media**

image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
Need no apply, not player facing

### CODE REVIEW

**Review by tom-leys — APPROVED** [review]
It's a one-line change to set a boolean to true. what could I possibly dislike?

**Comment by Equivocateur** [comment]
WHAT, HOW

File: Content.Server/Ghost/Roles/Components/GhostRoleComponent.cs [hunk 1]
```diff
@@ -63,6 +65,18 @@ public string RoleRules
...
+        [Access(typeof(GhostRoleSystem), Other = AccessPermissions.ReadWriteExecute)] // FIXME Friends
+        public bool RoleAntagonist
+        {
+            get => _roleAntagonist;
+            set
+            {
+                _roleAntagonist = value;
+                EntitySystem.Get<GhostRoleSystem>().UpdateAllEui();
+            }
+        }
```
Comment (metalgearsloth) [inline Content.Server/Ghost/Roles/Components/GhostRoleComponent.cs:78]: Why does this update EUI if it's not UI facing?

Also EntitySystem.Get<T> is obsolete.

**Comment by AJCM-git** [comment]
@Equivocateur you coming back to this?

**Comment by Equivocateur** [comment]
I'll reopen when I get to it

**Pull request closed without merging**